client/src/utils/DonuUtil.ts

```diff
@@ -15,9 +15,48 @@ export function donuToModel (donuModels: Donu.ModelDefinition[]): ModelInterface
       type: ModelInterfaceType.computational,
     } as ModelInterface;
   });
-}
+};
 
-export function donuSimulateToD3 (responseArr: Donu.SimulationResponse[]): SimulationVariable[] {
+export const aggregateMean = (dataColumn: number[]): number =>
```

Comment:
I think we might want to use this kind of function in other places so we might want to take it to some `DataUtil`?

Reply:
Why not just leverage d3 as it is loaded in the app anyway?
https://observablehq.com/@d3/d3-mean-d3-median-and-friends

Reply:
Or `lodash`, that's a good point.